contracts/README.md

```diff
@@ -115,14 +111,3 @@ points in order to demonstrate and test the flexibility we have.
   https://docs.rs/cosmwasm-std/0.13.0/cosmwasm_std/macro.create_entry_points_with_migration.html
 [cep]:
   https://docs.rs/cosmwasm-std/0.13.0/cosmwasm_std/macro.create_entry_points.html
-=======
-| Contract         | Has `query` | Has `migrate` |
```

Comment:
why is this document deleted?

Reply:
Because this document is same.
https://github.com/line/cosmwasm/blob/98dbb39b5d44b23c9cdd03a77dafe3051b343bd6/contracts/README.md#entry-points

Reply:
It's OK. I misunderstood. thank you.